queries.sql

```diff
@@ -8,4 +8,39 @@ SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered;
 SELECT * FROM animals WHERE name != 'Gabumon';
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+----
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+SELECT * FROM animals;
+UPDATE animals SET species = 'Pokemon' WHERE species is NULL;
+COMMIT;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals; 
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT S1;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+ROLLBACK TO S1;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
```

Comment:
- [ optional ] Hi you did a nice work. From the requirements, on line 35 you were supposed to update all animals in the table so there was no need for the condition `WHERE weight_kg < 0;` at this point